### PR TITLE
Adding a little Java Hello World app.

### PR DESCRIPTION
**Pull request by TB-EIT**

### CODE REVIEW

**Comment by BasiukTV** [comment]
Is this really necessary to be hosted in this repo?

File: hello_world.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+public class hello_world {
+
+  public static void main(String[] args) {
+    System.out.println("Hello World!");
```
Comment (BasiukTV) [inline hello_world.java:4]: Can we actually print out a greeting here, too?

File: hello_world.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  }
+}
```
Comment (BasiukTV) [inline hello_world.java:6]: Needs a new empty line at the end of the file.

**Review by BasiukTV — CHANGES_REQUESTED** [review]
Please address the code comments too.

File: hello_world.java (hunk 1 above)
Comment (TB-EIT) [reply hello_world.java:4]: This will be a part of another task.

**Comment by TB-EIT** [comment]
> Is this really necessary to be hosted in this repo?

Hey, this is just a Demo, relax a bit.

**Comment by TB-EIT** [comment]
Hey, @BasiukTV can you please take a look at this again, I did the requested changes.

**Review by BasiukTV — APPROVED** [review]
LGTM

File: hello_world.java (hunk 2 above)
Comment (TB-EIT) [reply hello_world.java]: Done.

**Review by BasiukTV — APPROVED** [review]
Yeah, sure, OK.

**Pull request merged**